### DISCUSSION TITLE
Fcitx5 input method doesn't show on a.17 version

### DISCUSSION THREAD

**Discussion by Ziqi-Yang** [zen-browser/desktop#2723] [General]
I've just updated my zen browser from `1.0.1-a.14` to `1.0.1-a.17`. However, with this update, I cannot use my fcitx5 input method on Zen browser. 

Since I'm using NixOS, a distro that has some difference from traditional linux distros like ArchLinux, I use https://github.com/0xc000022070/zen-browser-flake to install the Zen Browser application. I don't know if whether it is because Zen Browser itself or the NixOS build script causes this issue.

**Comment by Fxxxlei** [comment]
I haven't noticed any issues with fcitx5+Zen on Arch.

**Reply by CcMenta** [reply]
I also use arch and fcitx5 and works fine, so most likely this is a NixOS build script issue.
